daemon/mgr/container.go

```diff
@@ -541,6 +541,23 @@ func (mgr *ContainerManager) Get(ctx context.Context, name string) (*Container,
 		return nil, err
 	}
 
+	// get all execids belongs to this container
+	fn := func(v interface{}) bool {
+		execConfig, ok := v.(*ContainerExecConfig)
+		if !ok || execConfig.ContainerID != c.Key() {
```

Comment:
I think we can get the `Key` first and use the value. `c.Key()` will call the `lock` which is pretty annoy. WDYT?

Reply:
good advise :)